jgrapht-core/src/main/java/org/jgrapht/traverse/DepthFirstIterator.java

```diff
@@ -34,42 +36,13 @@
  * @param <V> the graph vertex type
  * @param <E> the graph edge type
  *
- * @author Liviu Rau
- * @author Barak Naveh
- * @since Jul 29, 2003
+ * @author Joris Kinable
  */
 public class DepthFirstIterator<V, E>
-    extends CrossComponentIterator<V, E, DepthFirstIterator.VisitColor>
+    extends CrossComponentIterator<V, E, DepthFirstIterator.SearchNodeData>
```

Comment:
Change to `extends CrossComponentIterator<V, E, DepthFirstIterator<V, E>.SearchNodeData>`.